completions/bash

```diff
@@ -192,6 +192,7 @@ __fastfetch_completion()
     local FF_OPTIONS_LOGO=(
         "-l"
         "--logo"
+        "--logo-file"
```

Comment:
This is a path option, not a logo option. Logo options are completed with the names of the build in logos.